src/graphql/2014/resolvers/featureResolver.ts

```diff
@@ -165,10 +165,21 @@ const Feature = {
       feature_specific.terrain_type_options &&
       'options' in feature_specific.terrain_type_options.from
     ) {
+      // Convert string values to proper StringOption objects
+      const processedOptions = feature_specific.terrain_type_options.from.options.map(option => {
```

Comment:
This feels like something that can be extracted into a general use method since we want this to work with any choice object that uses string.